test/tests/functional/pbs_trillion_jobid.py

```diff
@@ -444,3 +444,27 @@ def test_jobid_duplication(self):
         # Job should submit successfully because all existing id's has been
         # passed
         self.submit_job(lower=1, upper=2, job_id='3[]')
+
+    def test_jobid_resvid_after_multiple_restart(self):
+        """
+        Test to check the Jobid/Resvid should not wrap to 0 during
+        server restart multiple times consecutively either gracefully/abruptly
+        """
+        a = {'resources_available.ncpus': 15}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
+        self.set_svr_sv_jobidnumber(0)
```

Comment:
I don't think we need to reset svr_jobidnumber to zero in the database. In the system where this bug is present even without this step it is reproduced.

Reply:
Its necessary because i am checking the job sequence as well.

Reply:
I agree with @suresh-thelkar, Instead of setting it in DB, submit a job, get the job id and use that numeral to do further calculations

Reply:
Addressed!